common/wuwebview/wuwebview.cpp

```diff
@@ -921,6 +922,19 @@ void WuWebView::addInputsFromXml(const char *xml)
     addInputsFromPTree(pt.get());
 }
 
+IConstWUQuery* WuWebView::getEmbeddedQuery()
+{
+    if (!loadDll())
+        return NULL;
+
+    StringBuffer dllXML;
+    if (!getEmbeddedWorkUnitXML(dll, dllXML))
+        return NULL;
+
+    Owned<ILocalWorkUnit> embeddedWU = createLocalWorkUnit(dllXML.str());
```

Comment:
The good: this reuses the abstraction of existing code.
The bad: there is a much more efficient way of extracting this field from the xml.